Classes/Base/Context.php

```diff
@@ -330,4 +331,49 @@ protected static function addGroups( \Aimeos\MShop\Context\Item\Iface $context )
 
 		return $context;
 	}
+
+
+	/**
+	 * Adds the frontend date time to the context
+	 *
+	 * @param \Aimeos\MShop\Context\Item\Iface $context Context object including config
+	 * @return \Aimeos\MShop\Context\Item\Iface Modified context object
+	 */
+	protected static function addDateTime( \Aimeos\MShop\Context\Item\Iface $context )
+	{
+		// Handle the admin panel, according to the version number.
+		if ( version_compare( TYPO3_version, '9.2.0', '<' )
```

Comment:
Do we need those "if" conditions if we now use `isset()` for feature detection?

Reply:
I'm not sure, that the isset-detection is enough.

Following Scenario:
TYPO3 update from 7 to 9. I'm not sure, if an upgrade actually clears the uc settings.
The stuff from the uc in the backend user might be still there. Then you will get a specific frontend time in the shop, without any feedback about it, as long as these settings are stored there.

Fun fact:
The 9'er admin panel stores it's stuff in the TS settting of the backend user now.
`$GLOBALS['BE_USER']->getTSConfig()['admPanel.']`
So, you should be able to set a frontend time (for a specific user ?) there, but I have not tested this.

Reply:
OK, if you remove the conditions, we will merge it

Reply:
Does not work with 9.3, because the `TYPO3\CMS\Adminpanel\Service\ConfigurationService` is not yet available in the new admin panel.